forge-gui/res/cardsfolder/upcoming/the_lord_of_pain.txt

```diff
@@ -0,0 +1,10 @@
+Name:The Lord of Pain
+ManaCost:3 B R
+Types:Legendary Creature Human Assassin
+PT:5/5
+K:Menace
+S:Mode$ CantGainLife | ValidPlayer$ Player.Opponent | Description$ Your opponents can't gain life.
+T:Mode$ SpellCast | ValidActivatingPlayer$ Player | ActivatorThisTurnCast$ EQ1 | Execute$ TrigDealDamage | TriggerZones$ Battlefield | TriggerDescription$ Whenever a player casts their first spell each turn, choose another target player. CARDNAME deals damage equal to that spell's mana value to the chosen player.
+SVar:TrigDealDamage:DB$ DealDamage | ValidTgts$ Player | NumDmg$ X

```

Comment:
try `Player.!TriggeredPlayer`

Reply:
Should be !TriggeredActivator, I think, that's what's used in SpellCast triggers, right?

Reply:
yea probably better